web-server/routes/profile.js

```diff
@@ -0,0 +1,15 @@
+module.exports = app => {
+
+  app.get('/profile', (request, response, next) => {
+    const userHandle = request.user.handle
+
+    request.backOffice.getActiveLearners().then(users => {
+      const user = users.find(user => user.handle === userHandle)
```

Comment:
Most of the current users data should already be on `request.user` but if you need more data load it with `backoffice.getUserByHandle(request.user.handle)`